pkg/security/ebpf/c/exec.h

```diff
@@ -597,6 +666,38 @@ int kprobe_security_bprm_check(struct pt_regs *ctx) {
     return parse_args_and_env(ctx);
 }
 
+SEC("kprobe/setup_new_exec")
+int kprobe_setup_new_exec(struct pt_regs *ctx) {
+    struct syscall_cache_t *syscall = peek_syscall(EVENT_EXEC);
+    if (!syscall) {
+        return 0;
+    }
+
+    struct linux_binprm *bprm = get_linux_binprm_obj();
```

Comment:
Why fetching it from a map and override it with an argument ?

Reply:
ah, this is cruft I haven't deleted yet. thanks for catching it!